src/python/pants/option/config_test.py

```diff
@@ -2,23 +2,25 @@
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 from __future__ import annotations
 
-import unittest
 from dataclasses import dataclass
 from textwrap import dedent
 from typing import Dict
 
+import pytest
+
 from pants.engine.fs import FileContent
 from pants.option.config import Config, TomlSerializer
 
 
 @dataclass(frozen=True)
 class ConfigFile:
     content: str
-    default_values: Dict
+    unexpanded_default_values: Dict
+    expanded_default_values: Dict
     expected_options: Dict
 
 
-FILE_1 = ConfigFile(
+FILE_0 = ConfigFile(
```

Comment:
Renamed from FILE_1, FILE2 to FILE_0, FILE_1 to match the names that pytest assigns to the parameterized tests. It was very confusing when an error on what pytest calls "...file1" actually referred to our FILE_2...